src/main/java/com/nvs/store/dto/CartDTO.java

```diff
@@ -0,0 +1,70 @@
+package com.nvs.store.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Objects;
+
+
+@Builder
+public final class CartDTO {
+    @JsonProperty("id")
+    private final Long id;
+    @JsonProperty("actual")
+    private final Boolean actual;
+    @JsonProperty("cartItems")
+    private final List<CartItemResponse> cartItemResponses;
+    @JsonProperty("totalPrice")
+    private final BigDecimal totalPrice;
+
+    public CartDTO(
+            Long id,
+            Boolean actual,
+            List<CartItemResponse> cartItemResponses
+    ) {
+        this.id = id;
+        this.actual = actual;
+        this.cartItemResponses = cartItemResponses;
+        this.totalPrice = calculateTotalPrice();
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    private BigDecimal calculateTotalPrice() {
```

Comment:
suggest to extract this method to util class or service as its common in all cartDTOs and also important that scope of Data Transfer Object to have set data and transfer it - not make any business logic

Reply:
done ✅